helm/values-wcs-notprod.yaml

```diff
@@ -1,4 +1,6 @@
 ---
+hocs-generic-service:
 
-keycloak:
-  domain: wcs-notprod.homeoffice.gov.uk
+  keycloak:
+    realm: https://sso-dev.notprod.homeoffice.gov.uk/auth/realms/hocs-notprod
+    domain: wcs-notprod.homeoffice.gov.uk
```

Comment:
This would be a good time to be able to disable the CS specific cronjob.

Reply:
good shout.